package.nls.json

```diff
@@ -1,7 +1,7 @@
 {
     "extension.description": "An Azure Functions extension for Visual Studio Code.",
     "azFunc.refresh": "Refresh",
-    "azFunc.createFunctionApp": "Create Function App",
+    "azFunc.createNewProject": "Create New Project",
```

Comment:
How does it show up in the command palette?  Is it too generic?

Reply:
It shows up as "Azure Functions: Create New Project" so I think that's fine